src/components/dashboard/sidebar/CouponCodesSummaryCard.jsx

```diff
@@ -70,7 +78,13 @@ const CouponCodesSummaryCard = ({
           cardClassNames={className}
         >
           <p className="m-0">
-            <h3 className="float-left"> {couponCodesCount > 0 ? couponCodesCount : 0}</h3>{' '}<span className="ml-2">{COUPON_CODES_SUMMARY_DETAIL}</span>
+            <h3 className="float-left"> {couponCodesCount > 0 ? couponCodesCount : 0}</h3>{' '}
+            <span className="ml-2">{intl.formatMessage({
```

Comment:
Can we please use `FormattedMessage` here, same as above and change the id to 'enterprise.dashboard.sidebar.coupon.codes.summary.detail'